eth/vm/logic/stack.py

```diff
@@ -11,9 +11,9 @@ def pop(computation: BaseComputation) -> None:
 
 def push_XX(computation: BaseComputation, size: int) -> None:
     raw_value = computation.code.read(size)
-
-    if not raw_value.strip(b'\x00'):
```

Comment:
This strip was spending time to on a conversion that we might never need. Just let 0-bytes get pushed into the stack.